dependency-manager-tests/spm/SPMProject/MasterViewController.swift

```diff
@@ -0,0 +1,194 @@
+//
+//  MasterViewController.swift
+//  SPMProject
+//
```

Comment:
Same here - license header. Also, please run `./tools/license/check-license.sh` to see if all is good.